inventory.bak

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Do you plan on merging this? If so, it shouldn't be merged. Idea is you should be able to do
```
rake acceptance:provision_vms
rake acceptance:setup_pe
rake acceptance:setup_servicenow_instance[<fqdn>,<user>,<password>]
```

to get an `inventory.yaml` file with a real ServiceNow instance + PE Master. If `setup_servicenow_instance` doesn't have any args, then that will set up the mock ServiceNow instance container on the master VM.

Sorry, I should have added these instructions in the README but forgot to do that.

Reply:
sorry no I accidentally included this. I'll remove it.